content/hardware/04.pro/boards/portenta-x8/tutorials/09.image-flashing/content.md

```diff
@@ -82,14 +106,49 @@ To flash the Portenta X8, you need to begin by opening a terminal. Within the te
 uuu full_image.uuu
 ```
 
+If you have followed the __Flashing Mode without Carrier__ method to flash an OS or a custom image, the `uuu` command should be active and seeking for the board. While the process is active, please unplug and reconnect the USB-C® cable powering the Portenta X8 to allow entering the programming mode of the boot sequence. This should trigger the standing-by `uuu` task to run the flash process.
+
 When the flashing operation is finished, you should see a similar result as the following figure:
 
 ![uuu tool flashing success output](assets/uuu-flashing-success.png)
 
 Once you have verified it has successfully flashed the Portenta X8, the `BOOT` DIP switches that have been configured to the ON position, now need to be set to the OFF position. Otherwise, you will always have the Portenta X8 in Flashing mode whenever it is attached to a carrier. Recycle the power for Portenta X8 by reconnecting the board to your computer and start using with the latest updates.
 
+In case the Portenta X8 was flashed barebone, you will just need to recycle the power and should be ready with the latest OS image.
+
 ***After booting, you will need to wait 10 seconds until the Portenta X8 starts blinking Blue LED. The Blue LED indicates it was able to boot successfully.***
 
+### Portenta X8 Post-Flash Operation
```

Comment:
I don't think this part is relevant in the flashing tutorial and I'm worried that users start again thinking that they need a Portenta X8 Manager subscription (=foundries subscription) to be able to use X8 (something we are trying to avoid).

Reply:
Ah okay, that makes sense. Since it is an additional step, we can remove it without any issue and the users can still update to the latest OS image without this step. I'll remove to make sure we avoid the confusion :)